core/parameters/src/snapshots/near_parameters__config_store__tests__83.json.snap

```diff
@@ -0,0 +1,225 @@
+      "storage_remove_key_byte": 38220384,
+      "storage_remove_ret_value_byte": 11531556,
+      "storage_has_key_base": 54039896625,
+      "storage_has_key_byte": 30790845,
+      "storage_iter_create_prefix_base": 0,
+      "storage_iter_create_prefix_byte": 0,
+      "storage_iter_create_range_base": 0,
+      "storage_iter_create_from_byte": 0,
+      "storage_iter_create_to_byte": 0,
+      "storage_iter_next_base": 0,
+      "storage_iter_next_key_byte": 0,
+      "storage_iter_next_value_byte": 0,
+      "touching_trie_node": 16101955926,
+      "read_cached_trie_node": 2280000000,
+      "promise_and_base": 1465013400,
+      "promise_and_per_promise": 5452176,
+      "promise_return": 560152386,
+      "validator_stake_base": 911834726400,
+      "validator_total_stake_base": 911834726400,
+      "contract_compile_base": 0,
+      "contract_compile_bytes": 0,
+      "alt_bn128_g1_multiexp_base": 713000000000,
+      "alt_bn128_g1_multiexp_element": 320000000000,
+      "alt_bn128_g1_sum_base": 3000000000,
+      "alt_bn128_g1_sum_element": 5000000000,
+      "alt_bn128_pairing_check_base": 9686000000000,
+      "alt_bn128_pairing_check_element": 5102000000000,
+      "yield_create_base": 300000000000000,
+      "yield_create_byte": 300000000000000,
+      "yield_resume_base": 300000000000000,
+      "yield_resume_byte": 300000000000000
+    },
+    "grow_mem_cost": 1,
+    "regular_op_cost": 822756,
+    "vm_kind": "<REDACTED>",
+    "disable_9393_fix": false,
+    "storage_get_mode": "FlatStorage",
+    "fix_contract_loading_cost": false,
+    "implicit_account_creation": true,
+    "math_extension": true,
+    "ed25519_verify": true,
+    "alt_bn128": true,
+    "function_call_weight": true,
+    "eth_implicit_accounts": false,
+    "yield_resume_host_functions": false,
+    "limit_config": {
+      "max_gas_burnt": 300000000000000,
+      "max_stack_height": 262144,
+      "contract_prepare_version": 2,
+      "initial_memory_pages": 1024,
+      "max_memory_pages": 2048,
+      "registers_memory_limit": 1073741824,
+      "max_register_size": 104857600,
+      "max_number_registers": 100,
+      "max_number_logs": 100,
+      "max_total_log_length": 16384,
+      "max_total_prepaid_gas": 300000000000000,
+      "max_actions_per_receipt": 100,
+      "max_number_bytes_method_names": 2000,
+      "max_length_method_name": 256,
+      "max_arguments_length": 4194304,
+      "max_length_returned_data": 4194304,
+      "max_contract_size": 4194304,
+      "max_transaction_size": 4194304,
+      "max_length_storage_key": 2048,
+      "max_length_storage_value": 4194304,
+      "max_promises_per_function_call_action": 1024,
+      "max_number_input_data_dependencies": 128,
+      "max_functions_number_per_contract": 10000,
+      "wasmer2_stack_limit": 204800,
+      "max_locals_per_contract": 1000000,
+      "account_id_validity_rules_version": 1,
+      "yield_timeout_length_in_blocks": 200,
+      "max_yield_payload_size": 1024
```

Comment:
fwiw this seems to be missing the newly added state witness limits, so there's a change missing to a part of the code base here. Should be pretty easy to follow the thread from the test that generates this. `test_runtime_config_view` would be it.

Reply:
Thanks for the pointer!